src/index.ts

```diff
@@ -81,6 +93,24 @@ export type {
 	FieldState,
 } from "./fields";
 
+// Deprecations (unused import for @link references)
+import type {
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars
+	PhotoOEmbed,
```

Comment:
Do these types need to be imported? I'm guessing yes, but why is that? 😅

Reply:
For example, if you were to type the Vimeo oEmbed response which comes with other fields, you'd want to extend the `VideoOEmbed` type, not the `AnyOEmbed` one, for better accuracy :)

```typescript
type VimeoOEmbed = VideoOEmbed & {
	is_plus: string;
	account_type: "business" | "individual";
	duration: number;
	description: string;
	thumbnail_url_with_play_button: string;
	upload_date: string;
	video_id: number;
	uri: string;
};
```